slumber.yml

```diff
@@ -31,12 +31,20 @@ chains:
   select_value:
     source: !select
       message: Select a value
-      options:
+      options: !fixed
         - foo
         - bar
         - baz
         - a really really really really long option
         - "{{chains.username}}"
+  json_entry:
+    source: !prompt
+      message: JSON entry
+      default: '[1, "2", true]'
```

Comment:
Because `prompt`->`option` chains don't work in preview, I think this would be more representative if defined as a response chain. E.g.
```
dynamic_options:
  source: !response
    recipe: login
  selector: $.args
```